### PR TITLE
Remove (SearchResult|DocumentSummary)(Command|Reply) storage and docu…

### PR DESCRIPTION
**Pull request by toregge**
…mentapi messages.

@geirst : please review
@baldersheim : FYI

### CODE REVIEW

File: documentapi/src/main/java/com/yahoo/documentapi/messagebus/protocol/DocumentProtocol.java [hunk 1]
```diff
@@ -50,8 +50,8 @@ public class DocumentProtocol implements Protocol {
     public static final int MESSAGE_CREATEVISITOR = DOCUMENT_MESSAGE + 7;
     public static final int MESSAGE_DESTROYVISITOR = DOCUMENT_MESSAGE + 8;
     public static final int MESSAGE_VISITORINFO = DOCUMENT_MESSAGE + 9;
-    public static final int MESSAGE_SEARCHRESULT = DOCUMENT_MESSAGE + 11;
-    public static final int MESSAGE_DOCUMENTSUMMARY = DOCUMENT_MESSAGE + 14;
+    // public static final int MESSAGE_SEARCHRESULT = DOCUMENT_MESSAGE + 11;
```
Comment (geirst) [inline documentapi/src/main/java/com/yahoo/documentapi/messagebus/protocol/DocumentProtocol.java:53]: Please leave a comment on why these were removed.

File: documentapi/src/main/java/com/yahoo/documentapi/messagebus/protocol/DocumentProtocol.java [hunk 2]
```diff
@@ -70,8 +70,8 @@ public class DocumentProtocol implements Protocol {
     public static final int REPLY_CREATEVISITOR = DOCUMENT_REPLY + 7;
     public static final int REPLY_DESTROYVISITOR = DOCUMENT_REPLY + 8;
     public static final int REPLY_VISITORINFO = DOCUMENT_REPLY + 9;
-    public static final int REPLY_SEARCHRESULT = DOCUMENT_REPLY + 11;
-    public static final int REPLY_DOCUMENTSUMMARY = DOCUMENT_REPLY + 14;
+    // public static final int REPLY_SEARCHRESULT = DOCUMENT_REPLY + 11;
```
Comment (geirst) [inline documentapi/src/main/java/com/yahoo/documentapi/messagebus/protocol/DocumentProtocol.java:73]: Please leave a comment on why these were removed.

File: documentapi/src/vespa/documentapi/messagebus/documentprotocol.h [hunk 3]
```diff
@@ -55,9 +55,9 @@ class DocumentProtocol final : public mbus::IProtocol {
         MESSAGE_CREATEVISITOR       = DOCUMENT_MESSAGE + 7,
         MESSAGE_DESTROYVISITOR      = DOCUMENT_MESSAGE + 8,
         MESSAGE_VISITORINFO         = DOCUMENT_MESSAGE + 9,
-        MESSAGE_SEARCHRESULT        = DOCUMENT_MESSAGE + 11,
+        // MESSAGE_SEARCHRESULT       = DOCUMENT_MESSAGE + 11,
```
Comment (geirst) [inline documentapi/src/vespa/documentapi/messagebus/documentprotocol.h:58]: Please leave a comment on why these were removed.

File: documentapi/src/vespa/documentapi/messagebus/documentprotocol.h [hunk 4]
```diff
@@ -78,9 +78,9 @@ class DocumentProtocol final : public mbus::IProtocol {
         REPLY_CREATEVISITOR         = DOCUMENT_REPLY + 7,
         REPLY_DESTROYVISITOR        = DOCUMENT_REPLY + 8,
         REPLY_VISITORINFO           = DOCUMENT_REPLY + 9,
-        REPLY_SEARCHRESULT          = DOCUMENT_REPLY + 11,
+        // REPLY_SEARCHRESULT         = DOCUMENT_REPLY + 11,
```
Comment (geirst) [inline documentapi/src/vespa/documentapi/messagebus/documentprotocol.h:81]: Please leave a comment on why these were removed.

**Review by geirst — DISMISSED** [review]
👍  The removed messages have not been in use the last 10 years.

**Comment by geirst** [comment]
@bratseth please review as well.

**Review by baldersheim — DISMISSED** [review]
I assume they were just forgotten 10 years ago 👍

File: documentapi/abi-spec.json [hunk 5]
```diff
@@ -3293,4 +3207,4 @@
     ],
     "fields" : [ ]
   }
-}
+}
```
Comment (baldersheim) [inline documentapi/abi-spec.json:3210]: Missingeol at eof

**Review by geirst — DISMISSED** [review]
👍

**Review by baldersheim — APPROVED** [review]

**Pull request merged**